src/data/user/userServices.ts

```diff
@@ -1,3 +0,0 @@
-import { getCurrentUserMock } from './__mocks/userServicesMocks';
```

Comment:
Why did you delete this file? I don't see the reason to delete it

Reply:
My project has this file. It's weird.